packages/identifier/src/reserved.ts

```diff
@@ -1081,53 +1080,54 @@ const slurs = [
   'kafirs',
   'kike',
   'kikes',
-  'kraut',
-  'krauts',
+  'kkk',
+  'klukluxklan',
+  'muzzie',
+  'n1gga',
+  'n1gger',
+  'nazi ',
+  'negorid',
   'negress',
-  'negro',
-  'negros',
   'nig',
-  'nigs',
-  'nig-nog',
-  'nig-nogs',
+  'nigg3r',
+  'nigg4h',
   'nigga',
+  'niggah',
   'niggas',
+  'niggaz',
   'nigger',
+  'niggerachi',
+  'niggerette',
+  'niggerican',
+  'niggerino',
+  'niggeroid',
   'niggers',
+  'nigglet',
   'nigguh',
   'nigguhs',
-  'pajeet',
-  'pajeets',
+  'nig-nog',
+  'nig-nogs',
+  'nigs',
   'paki',
   'pakis',
+  'pedophile',
   'pickaninnie',
   'pickaninnies',
   'pickaninny',
   'pickaninnys',
   'raghead',
   'ragheads',
-  'retard',
```

Comment:
I disagree strongly. Having the ability to use this word in usernames is _not_ worth the tradeoff of making others feel unwelcome. I’d like this to be a community that prioritizes psychological safety, especially when there are other ways to say “delay” in multiple languages.

Reply:
Do you really think that 'explanation' applies to ALL the words removed? 
And it clearly does not explain anything about the words added. Do you believe those words have no meaning in any other languages?

Reply:
So, that totally explains why some of the other words are being removed too... /s I'm sure there are other uses of the word, but it's most known use is as an ableist slur. Remind me again why you're making this a hill to die on, fighting so hard for the use of a word to mean delay or slow down? Who's going to be hurt by the removal of this word? (No one). Who will be hurt by the USE of this word? (Lots of folks)

Reply:
> <img alt="Screen Shot 2023-07-13 at 18 38 43" width="574" src="https://user-images.githubusercontent.com/28465/253347479-533ce50f-9e47-4763-9502-35b739b2eacc.png">

If I wasn't allowed to use 'delay' as my username because it was a very common ableist slur in another language, I think I'd survive. Doesn't pass the smell test.

Reply:
Rather than this and its plural version being removed, -ed should be added. Not sure why this is an acceptable removal.

Reply:
Sorry but who are you to decide what basic words I can use or not in my own language?

Just because English is one of the most important languages in the World doesn't mean that anyone has the superiority to decide that there are 1st level and 2nd level languages. It underlies a quite cultural supremacist argument, tbh.

I'm pretty sure we can find better ways to avoid the use of Retard or Negro as a slut that don't involve blacklisting.

Reply:
@ferranrego You're completely off base. It has nothing to do with what language you are allowed to use. It's about creating a safe community. Again, as others here have said, whether or not there's another use of the word, it's not commonly used. In English, that word is most commonly used as a pejorative that is abelist and offensive. As Shane commented, I'll ask the same. Why is THIS the hill you choose to die on knowing it is, in fact, a commonly used slur?

Reply:
@ferranrego Oh don't worry, if somebody compiles a list of ethnic slurs used in other languages, we'll advocate for their inclusion here too.

Reply:
Sorry, @GabeWeiss, but me and hundreds of millions of people use these words everyday (because they are basic words).

We're not talking about the other N-words. We're talking about two edge cases that are really common words in Spanish, Catalan, Portuguese, Italian, French, Occitan... and dozens more.

Only because in English can be used in a peyorative way doesnt mena we must to backlsit them. There should be other ways to ensure that these words are not used as slurs. As I said, actively tracking these words for example and taking measures when needed.

The world doesn't end with English language. Nor is English superior to any other language.

There are more people in the World using "negro" or "retard" in its non-pejorative and everyday usage than there are people using it as slur.

Just take that into consideration too.

Reply:
@ferranrego Absolutely understand the point you're trying to make. The problem is, it doesn't MATTER if they're commonly used in other languages. This is for usernames. As you say, this is a global community. Folks in other languages should understand, that there are words that carry with them societal issues. I would argue the same thing if there were a commonly used English word that is a horrible slur in some other language. The discussion here is not around using the word within comments, for example, it's about usernames, and IMO (and clearly many others here) it's about a heightened level of accommodation to ensure a safe space.

Within a post, for example, you have the context of the larger comment to place the word in the proper framing. In a username, you don't. So, to your point, moderation is more difficult when you have to make a judgment call on a username without any further context. Solving the problem through more traditional moderation methods (natural language processing for sentiment for example) is more scalable.

I'm 100% with ya on not using this tool to ban general usage of the words because there are some other (good) alternatives to moderation to help with this sort of thing. But most of the established ways to scale this level of moderation fall over hard when we're talking about usernames specifically. The only way to properly moderate something like this would be human moderation, digging into each account in question to try to glean context around the username. And that's...not going to happen.

Reply:
Thanks for understanding my perspective and bringing the debate to something that is not white or black, I really appreciate it.

I agree with most parts of your points and I understand your argument.

But I also think that there is tech to live-track easily handles with the edge-case words and even filter them with different levels (of how potentially harmful they are). I'm sure, at least for now, would be doable with only some dozens of alerts per week.

Users can also receive a warning when creating the handle informing them that they are using a word used as a slur in other languages and that if they want to proceed they must be sure that their account has good faith and will be reviewed by the team.

The problem is that words like Negro are really basic, as you know is a color. And there are thousands of people with that surname and thousands of cases as names like companies, cultural magazines, organizations, civil rights associations, cooperatives, mountains, towns, cities, neighborhoods, lakes, social centers, streets, etc. Is not that easy as some people want to see it.

And also I think that if we put all languages at the same level (and we should) the amount of words blacklisted would be that big that would be almost impossible to get a allowed handle.

And, allow me a last comment, if we just accept that blacklisting these edge case words is the solution (temporary or not) we'll think that this is solved and the team will put the energy in the next stuff instead of tackling the problem with better solutions.

I hope I have explained myself and I hope no one feels attacked, if so I apologize. (Take also into account that I only learned English properly in the last 5 years)

Reply:
*nod* I can ALMOST guarantee that no one that works in this space ever feels like it's a solved problem. Ever. At least those that choose to work in the space rather than being voluntold to. :)

And I recognize, particularly the edge case negro for example, brings with it complications around common usage in Spanish. The decision points I'm seeing here are likely: 1) What's the moderation cost to handle the edge case, 2) How many users will be upset by not being able to use "Black" in Spanish in their username, 3) How many people will be offended by seeing "negro" pop up in their feeds?

They could likely do some user studies to test this theory. I will say, from my perspective the BEST way to test this theory while also creating a safe environment for as many users as they can, is to do what they're doing, and ban far and wide. Be as inclusive of any words you can find in as broad a swath as possible across all languages. Go how far you think you should go, then go farther.

When folks start chiming in "WTF?! Why did you ban xxxx?!" you can start to have some decent data around the impact because it's super explicit. It's much harder to get data the other way (not impossible certainly). But like, a user that leaves the platform (or just doesn't engage but doesn't leave) because they were tired of seeing slurs in usernames is harder to capture (unless you ask specifically in some kind of exit poll, but even then it's not likely you'll get large engagement because the person is already done with the platform).

Paramount is creating a safe space. In ALL languages. When you've gone too far and are at panic room level lockdown, you can ALWAYS back off and consider those edge cases at the other end once you've established some data around just how folks feel. But if your space isn't safe, you'll never find out cause folks aren't engaged with the platform.

Reply:
These are very interesting points and arguments to consider @GabeWeiss. I will think about them.

Thank you very much for taking your time to explain and sharing your takes!

Reply:
@ferranrego of course! Right back at ya. Glad it stayed civil. :D

Reply:
What about people who have the last name Montenegro or any institution from the country of Montenegro?

Reply:
> 

I'm not arguing that you can't use that word in your own language. I'll sideline "negro" for a bit, because I realize that it means "black" in Spanish. "Retard" on the other hand is very offensive and has very little use outside of some edge cases that you mentioned. Again, how many businesses are hurt because they can't use "retard" in their username? I'd argue very few. Yet, because it is allowed, we have "Capt Retard" as a username now. I find it very telling that your argument to have the freedom to use your language as you wish negates my need to not be called a "retard". I'm not sitting over here telling you that English is superior or that we should consider English swear words above other use cases. What you bring to the table, specifically for "retard" is a rare case, where it's easily abused. Beyond that, there are other words I'd rather not type here that have been removed that have ZERO use case outside of being a slur. Outside of the word "negro", I don't see any reason why any of the words that have been removed (especially "retard") have any clear and common use case in any language. You're taking a one-off situation and holding it up as the standard, claiming that I'm being English-superior, which I am not. I'm just saying that your use of certain words as an edge case does not negate my need for not being called a version of stupid that implies that people with intellectual disabilities should be killed. I know you don't like this being a good/bad case, but I'd rather not have people imply certain things in their usernames, which I think is more important that someone's "freedom" to use a certain word.

Reply:
I understand the linguistic perspectives in this discussion.  It's always been the case that a slur in one language may be a colloquial use in another.

But at the end of the day: 
1. Only the general population actually effected by a slur should get a say in its inclusion or exclusion. 

To put it simple - you don't know, and you can't know the full harmful effects caused by the word. You are not an expert.  No matter how much you try to justify that you are.  **_You cannot possibly make an informed decision_**.

2. If someone can't deal with the inability to use a few 5 character words in _usernames_, then they aren't capable of good faith reasonable discourse and are joining a social network with the _intent_ of acting in bad faith and showing that they have no desire to adhere to further community rules.  When invites are over, they'll just spam new accounts continuously with these words.